netdeploy/remote/deployedNetwork.go

```diff
@@ -63,10 +71,11 @@ type DeployedNetworkConfig struct {
 // DeployedNetwork represents the complete configuration specification for a deployed network
 type DeployedNetwork struct {
 	useExistingGenesis bool
-
-	GenesisData gen.GenesisData
-	Topology    topology
-	Hosts       []HostConfig
+	useBoostrappedFile bool
+	GenesisData        gen.GenesisData
+	Topology           topology
+	Hosts              []HostConfig
+	BootstrappedFile   BootstrappedFile
```

Comment:
maybe call it `BootstrappedNetwork`, since at this point, it's no longer a file ?